packages/web/src/components/organisms/TransactionForm/index.tsx

```diff
@@ -42,9 +43,12 @@ export const TransactionForm = ({ propertyAddress }: Props) => {
   const metricsAddress = useMemo(() => propertyAuthData?.property_authentication[0]?.metrics, [propertyAuthData])
   const checkWithdrawable = useCallback(
     () =>
-      Promise.all([getWithdrawalStatus(propertyAddress), getBlockNumber()]).then(([withdrawStatus, blockNumber]) =>
-        Boolean(withdrawStatus && blockNumber && withdrawStatus <= blockNumber)
-      ),
+      Promise.all([getWithdrawalStatus(propertyAddress), getBlockNumber()]).then(([withdrawStatus, blockNumber]) => {
+        return {
+          withdrawable: Boolean(withdrawStatus && blockNumber && withdrawStatus <= blockNumber),
+          remainBlocks: withdrawStatus || 0
```

Comment:
Is it correct...:thinking: ?

Is this value the number of blocks remaining? 

I haven't been able to check the actual numbers as I can't check the operation of the cancellation. I'm sorry:bow:

Reply:
Thank you for everything you do, despite the lack of our environment building!

In this part, `withdrawStatus` means the end of block number, therefore, the number subtracting `blockNumber` from `withdrawStatus` is the remaining blocks.

```ts
remainBlocks: withdrawStatus - blockNumber || 0
```

Also, the block number is incremented once every 13-15 seconds. So this process should be executed every 15 seconds. Polling, or by calling `web3.eth.subscribe('newBlockHeaders')` can be realized.

How hard can it be? Do you need a local environment before you can implement it?

Reply:
I understand!! I’ll fix implementation soon!

Reply:
Many thanks!